Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -274,6 +269,8 @@ open class MapView: UIView {
 
         // Set up managers
         setupManagers()
+
+        subscribeToMemoryWarningNotification()
```

Comment:
I would revert this change. We only do this once, so it doesn't seem necessary to create a method for it. Just leave it inline as it was before.

Reply:
Reverted